engine/src/main/java/com/agiletec/aps/system/common/entity/loader/AttributeRolesLoader.java

```diff
@@ -61,7 +70,32 @@ private void loadDefaultRoles(String attributeRolesFileName, Map<String, Attribu
 			_logger.error("Error loading attribute Roles : file {}", attributeRolesFileName, t);
 		}
 	}
-	
+    
+    public void loadLocalRoles(String configItemName, Map<String, AttributeRole> defaultCollection) {
+        try {
+            ConfigInterface configManager = this.getBeanFactory().getBean(ConfigInterface.class);
+			String xml = configManager.getConfigItem(configItemName);
+            if (StringUtils.isBlank(xml)) {
+                return;

```

Comment:
can a log at debug level with the value of `configItemName` be useful?

Reply:
Done